cmd/otel-allocator/server/bench_test.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 func BenchmarkServerTargetsHandler(b *testing.B) {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
`rand.Seed` is deprecated. If a custom seed is needed the global rand should not be used.